qiita_db/processing_job.py

```diff
@@ -850,6 +850,25 @@ def outputs(self):
                 name: qdb.artifact.Artifact(aid)
                 for aid, name in qdb.sql_connection.TRN.execute_fetchindex()}
 
+    @property
+    def processing_job_worflow(self):
+        """The processing job worflow
+
+        Returns
+        -------
+        ProcessingWorkflow
+            The processing job workflow the job
+        """
+        with qdb.sql_connection.TRN:
+            sql = """SELECT processing_job_workflow_id
+                     FROM qiita.processing_job_workflow_root
+                     WHERE processing_job_id = %s"""
+            qdb.sql_connection.TRN.add(sql, [self.id])
+            r = qdb.sql_connection.TRN.execute_fetchindex()
+
+            return (None if not r
```

Comment:
what about `return qdb.processing_job.ProcessingWorkflow(r[0][0]) if r else None`? I don't think the use of a negative conditional is necessary